### DISCUSSION TITLE
License Refund

### DISCUSSION THREAD

**Discussion by ghost** [aclap-dev/video-downloadhelper#344] [Help and Q&A]
### Your question

Hi, I bought a download helper premium today.
I need to download YouTube videos, so I should have bought it with a license for firefox, but I bought a license for chrome.
I want to refund my previously purchased license.

my email is : ryanoh99@gmail.com

Thank you

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: ko
license: accepted
key (only 16 first characters): 14bec788a26f4ea7
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Hegemony2",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
Regards,
jerome
